File: archiforge-ui/src/lib/api-v1-routes.ts
/**
 * Path segments for ArchiForge API v1 (no trailing slash; prefix with "/" when building URLs).
 * Keeps UI aligned with `ArchiForge.Api.Routing.ApiV1Routes` in tests.
 */
export const ApiV1Routes = {
  policyPacks: "v1/policy-packs",
  alertRules: "v1/alert-rules",
  alerts: "v1/alerts",
  compositeAlertRules: "v1/composite-alert-rules",
  alertSimulation: "v1/alert-simulation",
  alertTuning: "v1/alert-tuning",
  alertRoutingSubscriptions: "v1/alert-routing-subscriptions",
  digestSubscriptions: "v1/digest-subscriptions",
} as const;

File: ArchiForge.Api/Routing/ApiV1Routes.cs
namespace ArchiForge.Api.Routing;

/// <summary>Relative URL segments (no leading slash) for versioned operator/governance APIs — use with <c>"/" + ApiV1Routes.X</c> in tests and clients.</summary>
public static class ApiV1Routes
{
    public const string PolicyPacks = "v1/policy-packs";
    public const string AlertRules = "v1/alert-rules";
    public const string Alerts = "v1/alerts";
}
